packages/playground/client/src/index.ts

```diff
@@ -67,7 +67,11 @@ export interface StartPlaygroundOptions {
 	 */
 	onBeforeBlueprint?: () => Promise<void>;
 	mounts?: Array<MountDescriptor>;
-	shouldInstallWordPress?: boolean;
+	/**
+	 * Whether to install WordPress. Value may be boolean or 'auto'.
+	 * If 'auto', WordPress will be installed if it is not already installed.
+	 */
+	shouldInstallWordPress?: boolean | 'auto';
```

Comment:
How about this?

```suggestion
	shouldInstallWordPress?: boolean | 'if-not-installed';
```

Reply:
I like `if-not-installed` better than `auto` but is it really necessary to define a string default here? Can't we do a check against `undefined` to infer "auto" behavior & prevent mixing of type?

Reply:
@ashfame which of these reads clearer to you?

```ts
const config = { shouldInstallWordPress: auto ? undefined : false; };
// or 
const config = { shouldInstallWordPress: auto ? 'if-not-installed' : false; };
```

Reply:
Not sure I follow. What I meant was keeping `shouldInstallWordPress?: boolean` as it is. When that property is explicitly specified, do what's being asked (Current behavior) and when not defined at all, just assume the "auto" behavior i.e. install if no WP files are found, otherwise let it be.

Reply:
Sure, that makes sense and sounds convenient for omiting the `shouldInstallWordPress` option. I was just pointing out it reads awkward when you want to explicitly declare the `shouldInstallWordPress` key and need to explicitly type in `undefined` as a value.

Reply:
Yep, a user should never have to explicitly pass `undefined`. But I don't see why someone would have to do that with my suggestion? IF they need to specify it, they should know whether it's a true or false. And if they don't they should omit and let the default behavior take place, no?

Reply:
It's not uncommon to define object literals like this:

```ts
startPlaygroundWeb({
	shouldInstallWordPress: getUserPreference()
});
```

or like this:

```ts
startPlaygroundWeb({
	shouldInstallWordPress
});
```
or like this:

```ts
startPlaygroundWeb({
	shouldInstallWordPress: condition ? true : undefined
});
```

In which case the `undefined` literal would show up somewhere.

Reply:
Sure! Implementation details aside, I think we are on the same page as to how it should behave, which is: Do as explicitly asked and figure out if we need to, as the default behavior. Thought I will reiterate it, in case this PR was waiting on me to close the loop of our conversation. cc: @brandonpayton